scio-schemas/src/main/avro/schema.avsc

```diff
@@ -8,6 +8,7 @@
         {"name": "type", "type": "string"},
         {"name": "name", "type": ["null", "string"]},
         {"name": "amount", "type": "double"},
+        {"name": "bytes", "type": "bytes"},
```

Comment:
I don't think this is relevant to add the `bytes` field in this record

Reply:
agreed, this schema is used heavily in tests + scio-examples so let's not add this new field. IMO adding the bytes field to `TestRecord` should be a sufficient test case.

Reply:
@RustedBones @clairemcginty I agree with you both that the `bytes` field feels out of place in `Account`.

However, this PR was motivated by an issue we had where we could verify that a field of type `bytes` was set properly just before being serialized by `JobTest`, but nonetheless was empty after being read (turned out the issue was not related to Scio and we already fixed it).

I think there is value in a full round-trip test: verifying that values of all types can be serialized and read back again. If we remove the `bytes` field from `Account`, what do you suggest? Maybe we could modify `AvroInOutTest` to output `TestRecord` (or a variation thereof)

Reply:
`AvroInOutTest` is more a test example that a real test. This also does not check serialization, since it is using a test skink.
Since our AvroIO is mostly a wrapper around the beam `AvroIO` which is tested with this [schema](https://github.com/apache/beam/blob/243128a8fc52798e1b58b0cf1a271d95ee7aa241/sdks/java/core/src/test/avro/org/apache/beam/sdk/schemas/test.avsc) I think we can rely on their unit testing (and the avro lib one) ensuring that all types are supported.

However I'm not sure if `scio-schemas` is relevant anymore. Since`sbt-avro` 2.0.0, it is possible to define schemas in the module's test config. Projects could be more independent and have their own schemas matching their use case.